src/components/ProjectFormCreate/index.js

```diff
@@ -0,0 +1,186 @@
+import React, { Fragment } from 'react';
+import { Form, Message, Loader } from 'semantic-ui-react';
+import gql from 'graphql-tag';
+import { Query, Mutation } from 'react-apollo';
+import { Formik } from 'formik';
+
+import * as Yup from 'yup';
+import SelectInput from 'components/SelectInput';
+import TextInput from '../TextInput';
+import AsyncButton from '../AsyncButton';
+import { CenterWrapper } from '../../styles/App';
+import ErrorMessage from '../ErrorMessage';
+
+const ProjectSchema = Yup.object().shape({
+  title: Yup.string().required('Title is required'),
+  teamLeader: Yup.string().required('Team leader is required'),
+  members: Yup.array().min(1, 'Pick at least 1 member')
+});
+
+const getOptions = users => {
+  const options = [];
```

Comment:
another way is juse use map
`const options = users.map(user => ( {value: user.id, label: user.name })`